Improved wording. (#402)
sveltejs/learn.svelte.dev: content/tutorial/01-svelte/04-logic/05-keyed-each-blocks/README.md

title: Keyed each blocks
---

> By default, when you modify the value of an `each` block, such as adding or removing items, it will apply such modifications at the _end_ of the block, and then update any values that have changed. That might not be what you want.

Reviewer: ```suggestion
By default, when you modify the value of an `each` block, it will add and remove items at the _end_ of the block, and update any values that have changed. That might not be what you want.
```

I think this one reads better as is

> It's easier to show why than to explain. Click the 'Remove first thing' button a few times, and notice what happens: It doesn't actually remove the first `<Thing>` component, but rather the _last_ DOM node. Then it updates the `name` value in the remaining DOM nodes, but not the emoji, which in `Thing.svelte` is fixed when the component is created.

Reviewer: ```suggestion
It's easier to show why than to explain. Click the 'Remove first thing' button a few times, and notice what happens: it does not remove the first `<Thing>` component, but rather the _last_ DOM node. Then it updates the `name` value in the remaining DOM nodes, but not the emoji, which is fixed when each `<Thing>` is created.
```

I more closely aligned this with a previous change on the old tutorial
